test/parallel/test-http-response-multiheaders.js

```diff
@@ -47,8 +48,9 @@ const server = http.createServer(function(req, res) {
 });
 
 server.listen(0, common.mustCall(function() {
-  let count = 0;
-  for (let n = 1; n <= 2; n++) {
+  const runCount = 2;
```

Comment:
Can you move `runCount` to live outside of the `server.listen`?

Reply:
@maclover7 like this https://github.com/nodejs/node/pull/17419/commits/afacc3e0f081fb8e29fe667abd0ee9720fafb36f ?

Reply:
I believe so -- although the linter may want an extra blank line (you can check this via `make lint-js` :))

Reply:
@maclover7 it says `Running JS linter...` and then exits. Does it mean that everything is ok? :)

Reply:
@LEQADA Yep -- if no errors show up then all good. You can also see the linter CI job passing at https://ci.nodejs.org/job/node-test-linter/14156/